### ISSUE TITLE
Site domains different than test

### ISSUE THREAD

**Issue by marcosdipaolo** [laravel/valet#599]
At the beginning I wanted a `dev` domain name, then i found i had to issue a secure `https` to make it work and it all went well then. 
Now i see that for `vate share` i need the domain to be 'unsecure', then i think, ok, i'll set my domain to 'local' (just don't like test) and i'll make it unsecure and there appeared the problem:
Any unsecure (no valet secure) domain different that test just doesn't work. 
Anyone had the same?

**Comment by drbyte** [comment]
Yes. This is a known issue. Related: #138, #382, #148, #156, #504, #534.

I've found success using the code proposed in #630
